src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -556,30 +600,45 @@ handle_message({neighbor_accepted, Peer, Tag, Sender},
     {reply, ok, State};
 
 handle_message({neighbor_rejected, Peer, _Sender}, State) ->
+    %% @todo Is this necessary?
```

Comment:
I think this section is mainly here for logging that it attempted to peer and was rejected, but, we do need to disconnect the node because we had to open a connection to it to send the request, right?